### PR TITLE
Update to MUX 2.7

### LINKED ISSUE #7133: The rounded bottom corner on a tab doesn't get colored by the color picker
image

_enhance.gif_:
image

Probably regressed when we ingested the latest MUX that included the rounded corners. I'm _hoping_ that we don't need to get this updated in MUX itself, and we can just set some other key on the TabViewItem ourselves.

### LINKED ISSUE #8948: Saving the settings.json file creates a shadow on the tab?
hey so I'm getting something like this. every time i save the settings.json with wt in the background the tab changes to this:

image
which is a similar look

_Originally posted by @LuanVSO in https://github.com/microsoft/terminal/issues/7532#issuecomment-769471340_

### LINKED ISSUE #10508: Update to WinUI 2.6 (prerelease)
As title.

_migrie note:_
* If https://github.com/microsoft/microsoft-ui-xaml/pull/3769 is fixed before we ingest the latest MUX, check if #7133 and #8948 are fixed
* https://github.com/microsoft/microsoft-ui-xaml/pull/5145 may have also fixed #7133, but I think that's only in 2.7

### PR DESCRIPTION
**Pull request by zadjii-msft**
* this is the same thing as #10996, but with the fix that caused us to #11031  
* This includes https://github.com/microsoft/microsoft-ui-xaml/pull/3769, so we had to make some adjustments to how we handle tab colors. It works the same as before.
* Should enable #11231 to be started
* [x] Closes #10508
* [x] Closes #7133
* [x] Closes #8948
* [ ] I need to finish letting my 19H1 VM boot to make sure unpackaged still works

### CODE REVIEW

File: src/cascadia/TerminalApp/App.xaml [hunk 1]
```diff
@@ -20,7 +20,8 @@
         <ResourceDictionary>
             <ResourceDictionary.MergedDictionaries>
                 <!--  Include the MUX Controls resources  -->
-                <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls" />
+                <XamlControlsResources xmlns="using:Microsoft.UI.Xaml.Controls"
+                                       ControlsResourcesVersion="Version1" />
```
Comment (carlos-zamora) [inline src/cascadia/TerminalApp/App.xaml:24]: Curious: Why did we have to add the version here?

**Review by carlos-zamora — APPROVED** [review]
_**YES!**_ 🎉🎉🎉

File: src/cascadia/TerminalApp/App.xaml (hunk 1 above)
Comment (zadjii-msft) [reply src/cascadia/TerminalApp/App.xaml:24]: totally unfounded guess: this opts us out of the win11 styles, until we're ready to actually ingest them. Dustin did it in the original PR

**Comment by zadjii-msft** [comment]
Blocking because this explodes on 19H1 ☹️ 

@msftbot make sure @DHowett signs off on this

File: src/cascadia/TerminalApp/App.xaml (hunk 1 above)
Comment (DHowett) [reply src/cascadia/TerminalApp/App.xaml:24]: Correct.

**Review by DHowett — CHANGES_REQUESTED** [review]
f

**Review by DHowett — APPROVED** [review]
I wish this didn't explode our msixbundle by 10MB, but what can ya do

**Pull request merged**